### ISSUE TITLE
android device as sink

### ISSUE THREAD

**Issue by 257** [rom1v/usbaudio#2]
can it send audio from linux to android?
use case here is to send playing music on pc to android and have Shazam identify the song.

**Comment by rom1v** [comment]
> can it send audio from linux to android?

No. :(

**Comment by amosbird** [comment]
I wonder if there is a better way than using https://github.com/amosbird/SimpleProtocolPlayer  . Currently via the gnirehtet tunnel, the audio latency is around 1 second.

**Comment by rom1v** [comment]
@amosbird Several years ago, I implemented a "real-time" [talkie-walkie](https://groups.google.com/forum/#!topic/serval-project-developers/K-EH2RxtPbs) (https://github.com/rom1v/batphone/commit/3059bc4cb78c3f537f5fdfcfbd9a05a67a17b7c7) between Android devices on a Mesh network. A cache of 100ms was sufficient, so I guess you can get a latency far lower than 1 second on a "normal" network.

**Comment by amosbird** [comment]
Yes, the problem isn't related to network, but the quality of the audio stream. I think I might need a pulseaudio client written with https://github.com/google/oboe
